fix(SAI): Fix Stormwind citizens to be more Blizzlike in their behavior

LannyE opened the pull request:
Fixes:
Reginald Windsor is dead. I saw it with my own eyes - Removed
Adam and Billy - Talk, formation, and Waypoints
Removed Extra Guard just outside trade district
Justin, Brandon, and Roman - Talk, formation, and Waypoints
Miss Danna and students - Talk, formation, and waypoints - Add Geoff NPC
Donna and William - Formation and waypoint fix Closes #9144
Janey, Suzanne, and Lisan - Talk, formation, and waypoints
Orphanage/Matron/Shellenne - Waypoints and permanent placement
Karlee, Paige, Gil, and Fizzles - Talk, formation, and waypoints - Add Fizzles creature
Lil' Timmy - Talk, formation, waypoints, and despawn/disappear timer
Closes #6565
Underwater Construction Worker - Removed (gone in WOTLK)
Defias Prisoner and Guards - Formation fix Closes #10060
Sewer Beast - Creature Pool and spawn locations

These changes were made with as much info as I can gather from
wowhead, youtube and other sources.  May not be 100% Blizzlike.
Any existing waypoints were preserved with only slight modifications
due to bridge swallowing children and following NPCs getting stuck.
Some infomation was pulled from the Trinitycore database

This PR does not introduce the day and night cycle there is another PR I have coming with that info as Miss
Danna shouldn't be dragging her students out in the middle of the night to stare at statues and buildings
<!-- First of all, THANK YOU for your contribution. -->

## Changes Proposed:
-  See description
-  

## Issues Addressed:
<!-- If your fix has a relating issue, link it below -->
- Closes #9144
Closes #6565
Closes #10060

## SOURCE:
<!-- If you can, include a source that can strengthen your claim -->
Timmy's spawn time is not absolutely known.  I have read everythin from 90 minutes to 3 hours.  I randomized his reappearance between these times.
https://tbc.wowhead.com/npc=8666/lil-timmy#comments
Matron Nightingale is not only part of Children's Week.  She is always there.  Youtube video shows her outside Orphanage.
https://www.youtube.com/watch?v=s4nBCcq9qe4&ab_channel=ChaeriASMR
Also nothing in wowhead says she only spawns during children's week.
Reginald died in The Great Masquerade quest - He is long gone in WOTLK
https://wowwiki-archive.fandom.com/wiki/Reginald_Windsor

## Tests Performed:
<!-- Does it build without errors? Did you test in-game? What did you test? On which OS did you test? Describe any other tests performed -->
- Spent many days staring and following Civilians
- Merged fine in my environment


## How to Test the Changes:
<!-- Describe in a detailed step-by-step order how to test the changes -->

1. Merge
2. Look for the removed NPCs
3. Find the other NPCs and observe their behavior
4. look for Sewer Beast in one of his 6 locations

## Known Issues and TODO List:
<!-- Is there anything else left to do after this PR? -->

- [Timmy doesn't despawn he disappears.  This makes his kitten following him and the random reappearance more reliable - May have to revisit this later.]
- [ ]

<!-- If you intend to contribute repeatedly to our project, it is a good idea to join our discord channel. We set ranks for our contributors and give them access to special resources or knowledge: https://discord.com/invite/DasJqPba)
     Do not remove the instructions below about testing, they will help users to test your PR -->
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

Lynethris commented:
Tested and all seems good. (issues close check).

Few images:
<details>
![WoWScrnShot_032422_181557](https://user-images.githubusercontent.com/100411212/159982475-3a7943e9-b331-4c84-a242-e77b10c897ce.jpg)

![WoWScrnShot_032422_181244](https://user-images.githubusercontent.com/100411212/159982508-f6e63ab2-dd6c-4dba-a29a-93506c62336c.jpg)

</details>

Lynethris commented:
I had not reset the db, and I found a bug by chance. I have check with and without PR:

With:
![WoWScrnShot_032422_203824](https://user-images.githubusercontent.com/100411212/159997197-023e57c0-89aa-4724-88a0-23664835c4a1.jpg)

Without:
![WoWScrnShot_032422_203219](https://user-images.githubusercontent.com/100411212/159997217-6a9a5d62-57b7-4153-bf7a-c1b3e4f7351a.jpg)

LannyE commented:
Two words: Reginald Windsor.  The guy was so badass in the previous expansions removing him from the game rips the turrent off the steam tank and makes it lose its texture.  This was a good catch but to be honest this is not my SQL that had the error.  My SQL only uncovered this strange bug in a completely unrelated model.  If I remove Reginald's creature (guid: 86900) from the creature table the tank texture and turrent breaks.   Anyways, long story short I removed that tank modelid 25289 from the steam tank and replaced it with 25341.  Modelid 25289 is not being used anywhere else nor is the refurbished steam tank creature_template used by any other creature entity.  It seems to be working fine now.

acidmanifesto commented on data/sql/updates/pending_db_world/rev_1648089119541675100.sql:
```diff
@@ -0,0 +1,1194 @@
+(37063,0,0,0,38,0,100,0,1,0,0,0,0,29,0,50,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,1,0,38,0,100,0,2,0,0,0,0,29,0,160,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,2,0,38,0,100,0,3,0,0,0,0,29,0,200,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow'),
+(37063,0,3,0,38,0,100,0,4,0,0,0,0,29,0,310,0,0,0,0,12,2,0,0,0,0,0,0,0,'Stormwind City Guard - On Data Set - Set Follow');
```
need a line break at the end.

Nyeriah commented:
Could you provide sources for this? Because he was added to Stormwind during wrath with the removal of the onyxia chain. Before wrath he could only be found inside the cave. Sources say he was removed in 4.0x https://wowpedia.fandom.com/wiki/Reginald_Windsor

LannyE commented:
Reginald Windsor died in the quest The Great Masquerade when he was escorted back to Stormwind Keep to expose Onyxia.  I did this quest about 15 times maybe more on retail.  That quest was removed in 3.0.2.  https://wowwiki-archive.fandom.com/wiki/Quest:The_Great_Masquerade  The link from wowpedia does say he is alive in WOW but gives no info as to how or where he is at.  In Azerothcore his NPC is walking the waypoints that he took during the quest which shows it was a leftover or lifted from the quest.  Also the Trinitycore database does not have him as a creature entity anywhere.  I just can't find anything that says he survived somehow.  I dredged up this story from Google cache in 2013 https://webcache.googleusercontent.com/search?q=cache:mx8eAKJewMMJ:https://www.engadget.com/amp/2013-03-20-know-your-lore-marshal-reginald-windsor-and-the-great-masquerad.html+&cd=36&hl=en&ct=clnk&gl=us  and they do not talk about him surviving.  Everything else I find on the internet says he is gone and nothing has shown me a location he could have been at after TBC if he were alive.

Lynethris commented:
Work

acidmanifesto commented:
Still need that line break at the end of the sql file as we require all sql files to do so.

LannyE commented:
GIT and I are not friends.  I thought I had resolved it and my push said I was behind so I thought I was done.  I attempted another push with the space at the end.  Hopefully this worked.

LannyE commented:
Here is a picture of that guard that is showing in his screenshot from a different angle. 
![image](https://user-images.githubusercontent.com/65646022/160153555-ddf82828-1784-4b30-be19-3ed853c1b9d1.png)

acidmanifesto commented:
> Here is a picture of that guard that is showing in his screenshot from a different angle. 
> ![image](https://user-images.githubusercontent.com/65646022/160153555-ddf82828-1784-4b30-be19-3ed853c1b9d1.png)
> 
> 

Ah ok. Thanks for clarifying that.
